packages/create-commandkit/src/utils.ts

```diff
@@ -0,0 +1,50 @@
+import colors from 'colors';
+import gradient from 'gradient-string';
+import path from 'path';
+import url from 'url';
+
+const getDirname = (meta: ImportMeta) => path.dirname(url.fileURLToPath(meta.url));
```

Comment:
I do not see this being used elsewhere, so I think it makes sense to remove this function and instead declare __dirname directly